<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtDesigner 연습</string>
  </property>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>상태</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>시작</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>중지</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
